### PR TITLE
DOC: add gs read_parquet example

### LINKED ISSUE #6480: DOC: add example of reading parquet file from gs
https://docs.dask.org/en/latest/dataframe-create.html#reading-from-various-locations

Is fairly fleshed out which is good.

I think another example could be added at the end. My motivation here is that I often think you can't have too many examples (within reason). In addition, the example below is copy-pastable (although I believe it does require gcsfs to be installed but I think the error message makes this clear).

My proposal would be to change 

```
>>> df = dd.read_csv('s3://bucket-name/my-data-*.csv',
...                  storage_options={'anon': True})
```

to

```
>>> df = dd.read_csv('s3://bucket-name/my-data-*.csv',
...                  storage_options={'anon': True})
>>> df = dd.read_parquet("gs://dask-nyc-taxi/yellowtrip.parquet",
                     storage_options={"token": "anon"})
```

### PR DESCRIPTION
**Pull request by raybellwaves**
- [NA] Tests added / passed
- [NA] Passes `black dask` / `flake8 dask`

Closes #6480

### CODE REVIEW

File: docs/source/dataframe-create.rst [hunk 1]
```diff
@@ -76,6 +76,8 @@ storage backend.  You can do this with the ``storage_options=`` keyword:
 
    >>> df = dd.read_csv('s3://bucket-name/my-data-*.csv',
    ...                  storage_options={'anon': True})
+   >>> df = dd.read_parquet('gs://dask-nyc-taxi/yellowtrip.parquet',
+   ...                      storage_options={'anon': True})
```
Comment (quasiben) [inline docs/source/dataframe-create.rst:80]: Hmm, I thought the anonymous option for gcs was `{'token': 'anon'}`

**Comment by raybellwaves** [comment]
I'll check

File: docs/source/dataframe-create.rst (hunk 1 above)
Comment (raybellwaves) [reply docs/source/dataframe-create.rst]: Thanks for the spot.

**Comment by quasiben** [comment]
Thanks @raybellwaves !  Since this is a doc update I'll go ahead and merge now

**Pull request merged**